server/src/instant/core.clj

```diff
@@ -179,59 +179,59 @@
     (tracer/init)
 
     (log-init :uncaught-exception-handler
```

Comment:
Can you undo the whitespace changes in this file?

Your lsp should see that it's a macro, but maybe renaming `log-init` to `with-log-init` will fix it?

Reply:
with-log-init did the trick, thanks @dwwoelfel !